### PR TITLE
Join table visits

### PR DESCRIPTION
**Pull request by LionRouge1**
In this repo I :
- Created a table named Vets
- Created many-to-many between tables species and vets and between tables animals and vets

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -108,3 +108,99 @@ ON a.owner_id = o.id
...
+  GROUP BY A.species_id) as MS
+WHERE MS.Total_number = (SELECT MAX(MS.Total_number) FROM
+(SELECT A.species_id, COUNT(A.species_id) AS Total_number FROM vets Vt
+  LEFT JOIN specializations Sz ON Vt.id = Sz.vets_id
+  LEFT JOIN species Sp ON Sz.species_id = Sp.id
+  JOIN visits Vs ON Vt.id = Vs.vets_id
+  JOIN animals A ON Vs.animal_id = A.id
+  WHERE Vt.name = 'Maisy Smith'
+  GROUP BY A.species_id) as MS)) AS MS2
+  ON S.id = MS2.species_id;
```
Comment (uche-inyama) [inline queries.sql]: Kindly refer to this statement, use it modify your query you have written.

```
     SELECT species.name, count(*)
      FROM visits
      LEFT JOIN animals ON animals.id = visits.animal_id
      LEFT JOIN species ON animals.species_id = species.id
      LEFT JOIN vets ON vets.id = visits.vet_id
      WHERE vet.name = 'Maisy Smith'
      GROUP BY species.name
```

**Review by uche-inyama — CHANGES_REQUESTED** [review]
### Hi @LionRouge1 :slightly_smiling_face: 

Good job so far!

There are some issues that you still need to work on to go to the next project, we are almost there!




## Required Changes ♻️
_Check the comments under the review._

- [x] You did not populate the specializations table.
- [x] The vet's table was also not populated.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (LionRouge1) [reply queries.sql]: Ok, thank you very much @uche-inyama

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @LionRouge1 , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well implemented suggested changes.

### Optional suggestions :arrow_down_small: 

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**